chef/cookbooks/crowbar/templates/default/crowbar-router-migration.sh.erb

```diff
@@ -31,16 +31,20 @@ set -x
 mkdir -p $UPGRADEDIR
 rm -f $UPGRADEDIR/crowbar-router-migration-failed
 
-set +x
-source /root/.openrc
-set -x
+if [[ -f /usr/bin/neutron-ha-tool ]] ; then
```

Comment:
It probably means the script was executed on the wrong node. So maybe yes, failure seems more correct... but then the 1st commit is not needed as the failure will happen anyway.